app/common/locales/en.json

```diff
@@ -646,6 +647,7 @@
         "posts_total_map": "Displaying {{posts}} of {{total_nb}} posts with location information on the map.",
         "search_results": "Results:",
         "by" : "by",
+        "duplicate_option" : "There's a problem with the checkbox configuration in the survey because of duplicates, please fix it.",
```

Comment:
I think we should add "please notify the admin of the deployment" here instead since the users who adds post rarely are the same as the ones managing the deployments.

Reply:
This message would only be shown to admin. Does it. still makes sense to add that? I fear I can't totally understand what you meant by

> since the users who adds post rarely are the same as the ones managing the deployments.

Reply:
@renujain31 Ah, right, yes, if it is only shown to the admin its ok!